webapp/src/actions.js

```diff
@@ -12,6 +15,20 @@ export function connected(client) {
     };
 }
 
+export function userWantsToGiveFeedback(client) {
+    return (_, getState) => {
+        const currentUserId = getCurrentUserId(getState());
+        if (!currentUserId) {
+            return;
+        }
+
+        client.userWantsToGiveFeedback().then(({data}) => {
+            const channel = getChannel(getState(), data.channel_id);
+            navigateToChannel(getCurrentRelativeTeamUrl(getState()), channel.display_name);
```

Comment:
I think we want to use `channel.name` for navigation. It is what is used in the URL. feeback bot's username and display name happen to match (at least as of this PR and the last), but we should read it from `name`.
![display-name-vs-name](https://user-images.githubusercontent.com/13738432/153958099-c6079f8a-03df-45c8-b0fc-33b760a50ccb.png)

Reply:
For This comment and
> Could this be renamed to navigateToDM? With the @ in it, it will only work for DMs.

I must use `channel.name` as you indicated, but and then navigate to `{team}/channels/{channel.name}` for this to work, I'm being redirected automatically to the correct URL doing that 👍🏻

Reply:
> for this to work, I'm being redirected automatically to the correct URL doing that 👍🏻

Ahh, I think I told you slightly wrong; open and private channels behave like in the screenshot above and have the `name` property as the url. For direct channels like DMs with users and bots, it looks like display_name is empty, and the name is a concatenation of the user ids of the two parties, which works for redirection like you say. 0/5 on doing it as you have it, or we could possibly try reading the `channel.user_id`, looking up the user, and then constructing the `@{user.id}`.